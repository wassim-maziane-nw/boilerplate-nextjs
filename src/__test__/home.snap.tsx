import HomePage from "~app/page";

import { render } from "./test-utils";

const mockRouter = {
  push: jest.fn(),
};

jest.mock("next/navigation", () => ({
  useRouter: () => mockRouter,
}));

describe("Testing page example", () => {
  it("should render home", () => {
    const { container } = render(<HomePage />);
    expect(container).toMatchSnapshot();
  });
});
