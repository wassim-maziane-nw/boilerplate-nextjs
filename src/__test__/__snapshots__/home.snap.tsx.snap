// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing page example should render home 1`] = `
.emotion-0 {
  height: 100vh;
  width: 100vw;
  background-image: url('/assets/background.svg');
  -webkit-background-position: center;
  background-position: center;
  -webkit-background-size: 80%;
  background-size: 80%;
  background-repeat: no-repeat;
  background-color: #000000;
}

.emotion-1 {
  width: 8.625rem;
  height: 3.5rem;
  border-radius: 3.75rem;
  border-color: #404040;
  border-width: 1px;
  border-style: solid;
  background-color: #FFFFFF;
}

.emotion-1 span {
  font-family: "Neue Haas Grotesk Display Pro";
  font-weight: 600;
  font-size: 1rem;
  line-height: 1.5rem;
  color: #404040;
}

.emotion-1:hover {
  background-color: #EB514E;
}

.emotion-1:hover span {
  color: #FFFFFF;
}

<div>
  <div
    class="background-div emotion-0"
  >
    <div
      class="flex justify-center items-center h-full w-full"
    >
      <button
        class="emotion-1"
      >
        <span>
          rejoindre
        </span>
      </button>
    </div>
  </div>
</div>
`;
